engines/gob/dbase.h

```diff
@@ -36,12 +36,53 @@
 
 namespace Gob {
 
+/**
+ * A class for reading Multiple Index (.mdx) files for dBase.
+ * Currently, we only read the key definitions, not the actual index data.
+ *
+ */
+class dbaseMultipeIndex {
```

Comment:
```suggestion
class DbaseMultipeIndex {
```

All class names must start with a capital

Reply:
We already add a "dBase" class (named after the dBase III format), I just followed the same exception to the convention in this new class.